components/status-message/src/StatusMessage.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, createElement as h, ReactElement, ReactNodeArray } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+import { A } from "@not-govuk/components";
+
+import '../assets/StatusMessage.scss';
+
+export type StatusAction = {
+  /** Location to link to */
+  href: string
+  /** Text of the link */
+  text: string
+  /** Title of the link */
+  title?: string
+};
+
+export type StatusMessageProps = StandardProps & {
+  /** Summary of status */
+  status: string,
+  /** Links to perform status related actions */
+  actions?: StatusAction[]
+};
+
+export const StatusMessage: FC<StatusMessageProps> = ({
+    children,
+    classBlock,
+    classModifiers,
+    className,
+    status,
+    actions,
+    ...attrs
+  }) => {
+  const classes = classBuilder('hods-status-message', classBlock, classModifiers, className);
+
+  return status || children ? (
+    <div {...attrs} className={classes()}>
+      <p>
+        {status}
+        {children && <span className={classes('extra')}> {children}</span>}
+      </p>
+      {actions && actions.length &&
+        <ul className={classes('actions')}>
+          {actions.map((a, i) => (
+            <li key={i}>
+              <A className={classes('actions__button')} href={a.href} title={a.title}>{a.text}</A>
+            </li>
+          ))}
+        </ul>
```

Comment:
Any chance we could re-use AnchorList for this?: https://github.com/daniel-ac-martin/NotGovUK/tree/master/components-internal/anchor-list

Something like:
```jsx
  <AnchorList items={actions} classBlock={classes('actions')} />
```

...I think.

You'd have to change the way you applied styling to the buttons though...

It would allow you to get rid of the `StatusAction` type.

I don't know. Maybe it's not worth the bother. At least not right now.

Just food for thought.

Reply:
Yes, this is definitely a good shout. Not sure why I didn't do this originally, I think I had problems importing it :thinking: 
All good now though!